pkg/cmd/cli/cmd/newbuild.go

```diff
@@ -95,6 +96,8 @@ type NewBuildOptions struct {
 	Output        string
 	PrintObject   func(obj runtime.Object) error
 	LogsForObject LogsForObjectFunc
+
+	BuildArgs []string
```

Comment:
i'd treat it the same way we treat BinaryBuild (which is also specific to new-build).